processing/qrcode_extractor/requirements.txt

```diff
@@ -0,0 +1,2 @@
+opencv-python==4.8.0.76
```

Comment:
We should use the headless version of the library, i.e.``opencv-python-headless``.
Might be worth testing the extra modules ``wechat_qrcode`` in ``opencv-contrib-python-headless`` .